Python/instrumentation.c

```diff
@@ -712,12 +750,13 @@ instrument_per_instruction(PyCodeObject *code, int i)
         code->_co_monitoring->per_instruction_opcodes[i] = _PyOpcode_Deopt[opcode];
     }
     assert(code->_co_monitoring->per_instruction_opcodes[i] > 0);
-    *opcode_ptr = INSTRUMENTED_INSTRUCTION;
+    FT_ATOMIC_STORE_UINT8_RELAXED(*opcode_ptr, INSTRUMENTED_INSTRUCTION);
 }
 
 static void
 remove_tools(PyCodeObject * code, int offset, int event, int tools)
 {
+    Py_BEGIN_CRITICAL_SECTION(code);
```

Comment:
Do we really want `_Py_Instrument` to stop the world?  The reason why I didn't want to have that stop the world is because it can happen outside of calls to enable tracing, e.g. in `RESUME` which seems like it would be a massive slow down.